packages/expo-dev-menu/ios/DevMenuExtensionDefaultSettings.swift

```diff
@@ -10,10 +10,6 @@ class DevMenuExtensionDefaultSettings: DevMenuExtensionSettingsProtocol {
   }
 
   func wasRunOnDevelopmentBridge() -> Bool {
```

Comment:
i think this should maybe depend on whether or not the app is being served from a developer tool (or is a development mode bundle) rather than whether the build is DEBUG or not.

[i see this was always true even before your PR (except for the launcher's dev menu), so this is maybe a good follow up PR / task, just noticed it so wanted to point it out]

Reply:
ill be sure to make an issue for this before landing this pr